Topics/Development_Process/Standards_For_Best_Practices.md

```diff
@@ -0,0 +1,59 @@
+# Utilizing Standards to Adopt Best-Practices in Software Engineering
+
+## Table of Contents
+
+### [What Is a Standard?](#what-is-a-standard-1)
+
+### [Why Are Standards Important?](#why-are-standards-important-1)
+
+### [How Are Standards Created?](#how-are-standards-created-1)
+
+### [Why is This Relevant to CSC301 Students?](#why-is-this-relevant-to-csc301-students-1)
+
+### [Examples of Standards of Relevance](#examples-of-standards-of-relevance-1)
+
+### [Other Useful Standards](#other-useful-standards-1)
+
+### [Sources](#sources-1)
+
+### What Is a Standard?
+
+According to the Standards Council of Canada (SCC), a standard is defined "as a document, established by consensus and approved by a recognized body, that provides for common and repeated use, rules, guidelines or characteristics for activities or their results, aimed at the achievement of the optimum degree or order in a given context."
+
+### [Why Are Standards Important?](#why-are-standards-important-1)
+
+Broadly, standards play an important role for ensuring consistency, safety, and quality. They serve as guiding principles for excellence. In the world of technology, such as software engineering, they act as a benchmark for developers to create high-quality, maintainable code that meets industry standards and best practices.
+
+In addition, following standards promotes compatibility between different software components and systems, facilitating seamless integration and communication. This reduces development time, costs, and enhances the overall reliability and scalability of software solutions. This ensures that software systems meet industry-specific requirements and standards for safety, security, and data protection.
```

Comment:
Instead of saying "following standards", use the word "adopting" to appropriate terminology.